src/rexsdesktop/controller/User.java

```diff
@@ -518,7 +518,7 @@ public static boolean verificarPin(String pin, String correo) {
 
     public static boolean actualizarPerfilUsuario(String nombreCompleto, String email, int id) {
         Db db = new Db();

```

Comment:
same here about what I commented about injecting this as a dependency of the class